backend/service/auditsink/slack/slack.go

```diff
@@ -38,6 +44,19 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (service.Service,
 	return s, nil
 }
 
+// The struct is used in a Go template and the fields need to be exported
+type auditEventMetadata struct {
```

Comment:
nit: let's call it this or `auditTemplateData`
```suggestion
type auditEventTemplateData struct {
```

Reply:
updated to auditTemplateData